stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,123 @@
+
+**Please not that the restore process will deploy a new deployment with the filestore and SQL DB instance being restored to the backup specified.**
+
+Follow the steps below to ensure a smooth backup and recovery process.
+
+== Backup Process
+
+The backup process involves taking backup of Cloud SQL DB instance and filestore instance at a given point of time.
+
+=== Run backup playbook 
+
+The backup playbook require an active AAP on GCP foundation deployment to be running. The instructions below guide how run backup playbook.
+
+. Gather the following input variables
+.. *gcp_service_account_credentials_json_path*: Service account crendential json file path for google cloud account.
+.. *gcp_deployment_name*: Deployment name which you want to backup.
+.. *gcp_compute_region*: GCP region where foundation deployment deployed.
+.. *gcp_compute_zone*: GCP zone where foundation deployment deployed.
++
+Create an extra_vars.yaml file with the above parameters
++
+[source,bash]
+----
+gcp_deployment_name: "" 
+gcp_compute_region: ""
+gcp_compute_zone: ""
+gcp_service_account_credentials_json_path: "/secret/<service-account-creds-json-filename>"
+----
++
+. Run the backup playbook. Pull the docker image for operational container having tag with the version of the foundation deployment.
+  *Note*: Operational image tag should match with the version of your foundation deployment. For example, If your foundation deployment version is 2.2.1, you should pull operational image with tag 2.2.1
++
+[source,bash]
+----
+$ export IMAGE=<image-path>
+$ docker pull $IMAGE
+----
++
+. Run the backup playbook as container.
++
+Replace `absolute_path_google_application_credentials_dirname` with your path to service account credential json file for google cloud and `absolute_path_of_extra_vars_dirname` with the path to your directory where you extra_var file exists which you have created in step 1 of backup process. 
+Run the command to run the playbook.
++
+[source,bash]
+----
+$ docker run -v <absolute_path_google_application_credentials_dirname>:/secret -v <absolute_path_of_extra_vars_dirname>:/extra_vars $IMAGE redhat.onclouds.gcp_backup_deployment.yml -e @/extra_vars/<basename_of_extra_vars>
+----
++
+Make note of SQL DB Backup ID and SQL DB Backup name.
++
+
+
+
+=== Resources that are backed up
+
+Once the backup playbook run successfully, following resources are backup up.
+  - CLoud SQL DB instance
+  - Filestore instance
+
+== Restore Process
+
+The restore process will deploy a new deployment with the filestore and SQL DB instance being restored to the backup specified.
+
+*Note*: The restored filestore instance will use CIDR network range `192.168.244.0/29`, which is different from the default network filestore network range of the backup deployment.
```

Comment:
NOTE (in square brackets)
====
The restored filestore instance uses CIDR
====